n5-viewer/README.md

```diff
@@ -1,2 +1,78 @@
+
+Multiple compressors are not allowed. This property is required. 
+
+### blockSize
+A JSON array of positive integers indicating the length of each dimension of a chunk of the array. (Assumes the chunks are on a regular grid.) Must contain at least one item and cannot contain more than 4. This array must be the same length as `dimensions`, `pixelResolution`/`resolution`, `offset` (if present) and `downsamplingFactors` (if present), though this is not currently enforced by the schema. This property is required.
+
+### dimensions
+A JSON array of positive integers indicating the length of each dimension of the whole array. Must contain at least one item and cannot contain more than 4. This property is required.
+
+A couple of requirements are not currently enforced by the schema:
+1. The order of dimensions must be xyzt (or xyz, xyt, or xy). Dimension order is assumed by the software and is not expressed anywhere in the metadata, so a metadata document with incorrect dimension order will pass schema validation. 
+2. Channel is expressed in the hierarchy structure, not the metadata. Folders for each channel must be named e.g. c0/, c1/. 
+
+### pixelResolution
+A JSON object with exactly two properties:
+1. `unit`: The units of the spatial sampling interval. Can be any string. This property is required.
+2. `dimensions`: A JSON array containing the spatial sampling intervals for each dimension of the image. Can be any positive integer or float, including 0. Must contain at least one item and cannot contain more than 4. This property is required.
+
+This property is required for the n5 format.
+
+### resolution
+A JSON array containing the spatial sampling intervals for each dimension of the image. Can be any positive integer or float, or 0. Must contain at least one item and cannot contain more than 4. Neither allows nor assumes units. This property is required for the bigcat format. For the n5 format, use `pixelResolution`. 
+
+### downsamplingFactors
+A JSON array indicating the integer factor by which spatial resolution for each dimension was uniformly decreased, relative to the original image. Presumably, this happened in one of two ways: 
+1. Naive downsampling, in which data are uniformly discarded. For example, downsamplingFactors of 4 means every fourth sample was discarded along a particular dimension. 
+2. Average downsampling, in which subsets of samples are averaged to produce a new, smaller array. For example, downsamplingFactors of 4 means every four samples were averaged to produce a new sample. If dimensions are downsampled evenly, the total number of samples is reduced by a factor of N^M, where N is the number of dimensions and M is the downsampling factor.   
+
+A few requirements are not currently (or not yet) enforced by the schema:
+1. s0/ should have `downsamplingFactor`s of 1 (e.g. [1,1] for a 2-dimensional image), since s0 should be the original image. 
```

Comment:
I agree it would be nice for consistency for `s0` to have `downsamplingFactor` of `[1,...,1]`. You mention it's not enforced now. I think in practice it is usually not present at all for `s0`. Not sure if we want to claim that it should be, or not.

Reply:
I'm also not sure, I don't really care.  

The small part of me that does care likes that it's not present for `s0`.  Having the parameter there implies that it could have been something other than `[1...,1]`, and I don't like that.